src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -661,6 +662,7 @@ private void AddTypeKind(INamedTypeSymbol symbol)
                 {
                     switch (symbol.TypeKind)
                     {
+                        // FindValidCloneMethod is called instead of symbol.IsRecord since we want records in VB compilations to display as records.
```

Comment:
Not sure why this would be the case: I would expect a PENamedTypeSymbol from VB to return true for this as well. It would be very odd to me, imo, if this was the observable behavior:

1. In C#, I define a record type. ITypeSymbol.IsRecord is true
2. In C#, I reference the dll from the original project. ITypeSymbol.IsRecord is true.
3. In VB, I reference the dll from the original project. ITypeSymbol.IsRecord is false.

This is pretty inconsistent behavior, I'd expect scenarios 2 and 3 to have the same result. @AlekseyTs, what are your thoughts on this?